crates/matrix-sdk-ui/src/timeline/mod.rs

```diff
@@ -455,20 +456,22 @@ impl Timeline {
         })
     }
 
-    /// Given a transaction id, try to find a remote echo that used this
-    /// transaction id upon sending.
-    async fn find_remote_by_transaction_id(&self, txn_id: &TransactionId) -> Option<OwnedEventId> {
+    /// Returns a local or remote timeline item identified by this transaction
+    /// id.
+    async fn any_timeline_item_by_txn_id(
```

Comment:
`find_item_by_transaction_id` might be a clearer name. The "any" part doesn't seem to add anything, although I understand it makes sense in the context that this method used to find only remote items.

Reply:
There's already `item_by_transaction_id` which returns only local echoes, so I wanted something to distinguish it. But maybe the right move would be to rename `item_by_transaction_id` to something else :thinking:

Reply:
Yeah, I would go for `local_item_by_transation_id` and `item_by_transaction_id` I guess.